README.md

```diff
@@ -15,9 +15,9 @@ Role Variables
 
 All role variables are displayed with their default value.
 
-Specify the version of PHP you want to use (`7.0`, `7.1` or `7.2`):
+Specify the version of PHP you want to use (`7.0`, `7.1`, `7.2` or `7.3`):
 
-    php7_version: 7.1
+    php7_version: 7.3
```

Comment:
Do we want to switch the default to 7.3 already?

Reply:
It is an LTS already, so why not :-)

Reply:
I just noticed that I did not change the default by the way, only the default in the readme 🙈

Reply:
Fine by me :slightly_smiling_face: 

I do consider it a BC-break, so let's bump the major version!